guides.md

```diff
@@ -0,0 +1,83 @@
+
+> Protocols are the "attitudes" or "the manner in which one approaches each and
+every element in our space." They are different than pure rules or
+instructions; they are ways that we establish order and maintain practices
+across our group [...] -- They define the way we ought to proceed or behave in
+different situations. As such, they are normative, or premised on values,
+morals, and an idea of how things ought to be done. They are a manifestation of
+our values.
+
+## Scheduling a meeting
+
+Note: Consider whether this meeting might be a global interest to other
+members. Skip below steps at your discretion for low-stakes topics.
+
+- Try to announce your intention to schedule a meeting on a topic, and ask
+  who's interested to participate. Allow a day for people to respond.
+- Consider who might be most interested, and what timing might work for
+  them.
+  - Check the [monthly `availability` calendar for travel/vacation plans][availability].
+  - Check everyone's [weekly availability `schedules`][schedules].
+- If 2-3 candidate timeslots feel obvious from looking at internal
+  availabilities:
+  - Reserve them all in [Hypha calendar][calendar] immediately, for example: `HOLD:
+    Infrastructure Meeting (Option 1)`
+  - Ask in chat whether anyone who wants to attend has any blocks on any
+    of the options.
+- If short-notice or speed of booking is important:
+  - Claim an open HOLD [:information_source:](/meetings.md#holds) slot.
+    - If time permits or topic isn't an all-hands matter, consider
+      leaving the hold available for someone else.
+    - If claiming a hold, edit the calendar immediately, and announce
+      meeting in chat.
+- If your must find a new timeslot:
+  - Create a new poll. (We currently use [When2Meet](https://www.when2meet.com/).)
+  - Leave ample time for completion (ideally 1 week).
+  - Check on non-responders in following days, and send (gentle)
+    reminders as needed.
+
+## Hosting a meeting
+
+- **Before** the meeting...
+  - create a stub agenda as soon as possible (can be done before scheduling)
+    - Hint: use [an agenda `template`][template]
+  - add the agenda to the [`meetings` index][meetings]
+  - update the [organizational calendar][calendar] as event scheduling
+    progresses
+  - mention the meeting time in chat, and link the agenda.
+  - encourage migration of related actions, decisions and discussions
+    into agenda from chat
+- **During** the meeting...
+  - ask for agenda items (start of meeting)
+  - facilitate discussion
+  - keep time
+  - call attention to action items, upcoming decisions and discussions
+- **After** the meeting...
+  - add important dates/reminders to calendar
+  - review and merge notes
+  - update [meeting index][meetings] as best location changes
+    (hackmd => review (PR) => file)
+  - as needed, set up loomio threads and decisions
+  - as needed, send reminders of action items
+  - future: migrate action items into task tracker
```

Comment:
Unclear how "future" is being used here, can we just get rid of "future" given task tracking immanent arrival?

Reply:
Good point :)

Reply:
```suggestion
  - migrate action items into task tracker
```